windows/setup_python_deps.bat

```diff
@@ -1,6 +1,5 @@
 set PYTHONPIP=C:\Python27\Scripts\pip.exe
 
-%PYTHONPIP% install python-2.7.9\Django-1.6.11-py2.py3-none-any.whl
 %PYTHONPIP% install python-2.7.9\matplotlib-1.4.3-cp27-none-win_amd64.whl

```

Comment:
It works works with 2.7 64bit MSI installer https://www.python.org/downloads/release/python-2711/ on 2012 and 2008 server. According to http://stackoverflow.com/questions/4750806/how-to-install-pip-on-windows it is supported since 2.7.9 (although I haven't tried lower versions)